controller/label.go

```diff
@@ -149,3 +147,22 @@ func ConvertLabels(appl application.Application, request *http.Request, labels [
 	}
 	return ls
 }
+
+// ConvertLabelsSimple converts a array of Label IDs into a Generic Reletionship List
+func ConvertLabelsSimple(request *http.Request, labelIDs []interface{}) []*app.GenericData {
+	ops := []*app.GenericData{}
```

Comment:
you could initialize an array with the length of `labelIDs`

Reply:
yes! done in https://github.com/fabric8-services/fabric8-wit/pull/1627/commits/20a09410a4cb30b338229bbad7a31ddf4431f377